pymc_extras/distributions/discrete.py

```diff
@@ -509,7 +509,7 @@ def dist(cls, r, alpha, time_covariate_vector=None, *args, **kwargs):
         time_covariate_vector = pt.as_tensor_variable(time_covariate_vector)
         return super().dist([r, alpha, time_covariate_vector], *args, **kwargs)
 
-    def logp(value, r, alpha, time_covariate_vector=None):
+    def logp(value, r, alpha, time_covariate_vector):
         if time_covariate_vector is None:
```

Comment:
Seems like your logp doesn't handle ndim > 1 right? In that case raise NotImplementedError if value.ndim > 1 ?